go/inst/instance_dao.go

```diff
@@ -198,6 +198,16 @@ func (instance *Instance) checkMaxScale(db *sql.DB, latency *stopwatch.NamedStop
 	return isMaxScale, resolvedHostname, err
 }
 
+// RegexpMatchPattern returns true if s matches any of the provided regexpPatterns
+func RegexpMatchPatterns(s string, regexpPatterns []string) bool {
+	for _, filter := range regexpPatterns {
+		if matched, err := regexp.MatchString(filter, s); err == nil && matched {
+			return true
+		}
+	}
+	return false
+}
+
```

Comment:
please move this function somewhere more generic, such as `go/inst/instance_utils.go`